dashboard/config/routes.rb

```diff
@@ -271,5 +271,6 @@ def redirect_to_teacher_dashboard
 
   get '/api/section_progress/:section_id', to: 'api#section_progress', as: 'section_progress'
   get '/api/student_progress/:section_id/:student_id', to: 'api#student_progress', as: 'student_progress'
+  get '/api/user_progress/:script_name', to: 'api#user_progress', as: 'user_progress'
```

Comment:
i can't figure out if/where this is used in this PR, or is it setup for future work?